### PR TITLE
Refactoring ONOS topo CLI

### PR DESCRIPTION
**Pull request by tomikazi**

### CODE REVIEW

File: pkg/topo/create.go [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+}
+
+func getCreateKindCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "kind <id> <name> [args]",
+		Args:  cobra.MinimumNArgs(2),
+		Short: "Create Kind",
+		RunE:  runCreateKindCommand,
+	}
+	cmd.Flags().StringToStringP("aspects", "a", map[string]string{}, "default aspect for entities of this kind")
```
Comment (kuujo) [inline pkg/topo/create.go:59]: Is the value here expected to be a raw JSON blob?

Comment (tomikazi) [reply pkg/topo/create.go:59]: Yes.

Comment (tomikazi) [reply pkg/topo/create.go:59]: It should be "aspect" though; typo.

Comment (tomikazi) [reply pkg/topo/create.go:59]: I can also add the --set type.attribute=value usage as with update.

File: pkg/topo/create.go [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+func createObject(object *topoapi.Object, cmd *cobra.Command) error {
+	aspects, err := cmd.Flags().GetStringToString("aspects")
+	if err != nil {
+		return err
+	}
+
+	// Apply all aspect values
+	for aspectType, aspectValue := range aspects {
+		err := object.SetAspectBytes(aspectType, bytes.NewBufferString(aspectValue).Bytes())
```
Comment (kuujo) [inline pkg/topo/create.go:111]: You can actually just `[]byte(aspectValue)` cast the `string` to `[]byte`

File: pkg/topo/create.go (hunk 1 above)
Comment (kuujo) [reply pkg/topo/create.go]: Yes I agree it should be singular since each instance of the flag is a single aspect

File: pkg/topo/create.go (hunk 2 above)
Comment (tomikazi) [reply pkg/topo/create.go:111]: Let me try...

File: pkg/topo/get.go [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+
+	default:
+		_, _ = fmt.Fprintf(writer, "\n")
+	}
+}
+
+func printAspects(writer io.Writer, object topoapi.Object, verbose bool) {
+	if verbose {
+		for aspectType, aspect := range object.Aspects {
+			_, _ = fmt.Fprintf(writer, "\t%s=%s\n", aspectType, bytes.NewBuffer(aspect.Value).String())
```
Comment (adibrastegarnia) [inline pkg/topo/get.go:204]: Is this will be printed in JSON format?

Comment (adibrastegarnia) [reply pkg/topo/get.go:204]: I mean when we have more complex aspects to print, we will see them in a well formed JSON?

Comment (tomikazi) [reply pkg/topo/get.go:204]: Yes.

**Review by adibrastegarnia — APPROVED** [review]

**Pull request merged**